west/cli.py

```diff
@@ -21,7 +21,7 @@
 
 CONTEXT_SETTINGS = {
     "help_option_names": ["-h", "--help"],
-    "default_map": settings.get_config_settings(),
+    "default_map": settings.get_config_settings("west.ini"),
```

Comment:
Could be nice to load at several places, including `/etc/west.ini`, `~/.west.ini` and, yes, `./west.ini` and `./.west.ini`

Reply:
Click has a helper for that: http://click.pocoo.org/6/utils/#finding-application-folders

Reply:
Nice ! Nothing for /etc/, though, it seems.